FluidNC/src/Configuration/GenericFactory.h

```diff
@@ -8,6 +8,22 @@
 #include "HandlerBase.h"
 
 namespace Configuration {
+    template <typename BaseType>
```

Comment:
Had to make BuilderBase a non-inner class to be able to forward declare it in HandlerBase.h